README.md

```diff
@@ -1,5 +1,95 @@
 # ActiveRecord CRUD Intro
 
+## Objectives
 
-1. Explain Create, Read, Update, Delete 
-2. Add codesnippets to the explanations as examples
\ No newline at end of file
+
+1. Understand CRUD - Create, Read, Update, and Delete.
+2. Get familiar with how to use ActiveRecord methods for Crud.
+
+## What is CRUD?
+
+CRUD is an acronym for the four verbs we use to operate on data: **C**reate, **R**ead, **U**pdate and **D**elete. Active Record automatically creates methods to allow an application to read and manipulate data stored within its tables.
+
+### Create
+
+To create new objects, we can either use the `new` or `create` method. 
+
+The `new` method will create a new object without any attributes.
```

Comment:
This is a bit confusing because you can use `.new` with attributes, it just doesn't save to the database. For example `cat = Cat.new(:name => "Maru", age => 12)` would be okay. 

What about updating this to "The `new` method instantaites a new object in memory. In this case, we instantiate it without any attributes.